src/components/study-container.js

```diff
@@ -520,30 +466,125 @@ export function StudyContainer({ view, onChangeTab }) {
         // Note: studyUuid and dispatch don't change
     }, [studyUuid, dispatch, snackError, snackWarning]);
 
-    //handles map manual mode network reload
-    useEffect(() => {
-        if (mapManualRefresh && forceReloadNetwork) {
-            loadNetwork(true);
-        }
-    }, [forceReloadNetwork, loadNetwork, mapManualRefresh]);
-
     useEffect(() => {
         if (studyUuid) {
             loadTree();
         }
     }, [studyUuid, loadTree]);
 
+    function parseStudyNotification(studyUpdatedForce) {
```

Comment:
This is duplicated with network map. We should either:
 - share code with a common function
 - maybe better centralize the processing of the event and then put in the store results (substations ids, etc) which is then used by components

Reply:
I opted for centralizing the notification processing, it simplifies the code on the side of the network map tab